yarn.lock

```diff
@@ -447,6 +572,24 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@babel/parser@npm:^7.20.1":
+  version: 7.20.1
+  resolution: "@babel/parser@npm:7.20.1"
+  bin:
+    parser: ./bin/babel-parser.js
+  checksum: 2db7ba692b8b3054df129876b115ed8b265d5c138dee5e1db56a999209df3dd3d508e0985bf3cc44fc432498da094c164906b531d049608e55e208dae0678d42
+  languageName: node
+  linkType: hard
+
+"@babel/parser@npm:^7.20.2":
```

Comment:
couldn't we consolidate these two version? (the one above seems to be different only by path number)